Dockerfile

```diff
@@ -6,6 +6,8 @@ RUN apk -v --update add \
         certbot \
         bash \
         && \
+    apk add --no-cache --virtual openssl-dev qt-dev openssl \
```

Comment:
this can probably be simplified by just adding 
```
openssl-dev \
qt-dev \
openssl \
```
after `bash`.

Also, are the `openssl-dev` and `qt-dev` packages both necessary?

Reply:
It appears that they are both necessary for installing `openssl`.  I could not install `openssl` without those 2.